### PR TITLE
Add kernelfactory.canonical.com to releases page

### PR DESCRIPTION
**Pull request by pedoch**
# Done

Add kernelfactory.canonical.com to the releases page

### CODE REVIEW

**Review by petesfrench — APPROVED** [review]
LGTM

File: webapp/releases.py [hunk 1]
```diff
@@ -141,6 +141,11 @@
         "staging_domain": "library.staging.canonical.com",
         "repository": "canonical-web-and-design/canonical",
     },
+    {
+        "production_domain": "kernelfactory.canonical.com",
+        "staging_domain": "staging.kernelfactory.canonical.com",
+        "repository": "canonical-web-and-design/kernelfactory.canonical.com",
```
Comment (anthonydillon) [inline webapp/releases.py:147]: Isn't this `canonical` org now?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
LGTM one comment on the code

**Comment by pedoch** [comment]
Hi @anthonydillon, I've changed the GitHub organization name, could you take a look when you can?

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**